test_conformance/math_brute_force/function_list.cpp

```diff
@@ -291,13 +291,12 @@ const Func functionList[] = {
     ENTRY(erfc, 16.0f, 16.0f, 4.0f, 4.0f, FTZ_OFF, unaryF),
     ENTRY(erf,  16.0f, 16.0f, 4.0f, 4.0f, FTZ_OFF, unaryF),
 
-    // relaxed error is overwritten in unary.c as it is 3+floor(fabs(2*x))
+    // 3+floor(fabs(2*x)) is added to the relaxed error in unary.c
```

Comment:
```suggestion
    // floor(fabs(2*x)) is added to the relaxed error in unary.c
```

Same below for exp2.